runtime/rastrace/mt.tdf

```diff
@@ -59,3 +59,6 @@ TraceExit-Exception=Trc_MethodExitNXS Overhead=1 Level=5 Group=throw,nativeMetho
 
 TraceEvent=Trc_MethodReturn Overhead=1 Level=5 Group=methodArguments Template="return value: %s"
 TraceEvent=Trc_MethodException Overhead=1 Level=5 Group=methodArguments Template="exception: %s"
+
+TraceEvent=Trc_MethodArgumentsStr Overhead=1 Level=5 Group=methodArguments Template="this: (String)@%p - %s method arguments: (%s)"
+
```

Comment:
Just one final newline, not two, please.

Reply:
The blank line before line 62 was fine, but optional. The file must end with one newline, not two: there should be a newline at the end of line 62 but not another that ends line 63.